webrender/build_gfx.rs

```diff
@@ -392,9 +404,27 @@ fn replace_non_sampler_uniforms(new_data: &mut String) {
     );
 }
 
+fn get_set_from_line(code: &Vec<&str>) -> usize {
+    let (sampler_name, _) = code.split_last().unwrap();
+    match sampler_name.as_ref() {
+        "sColor0" |
+        "sColor1" |
+        "sColor2" |
+        "sPrevPassAlpha" |
+        "sPrevPassColor"  => return DESCRIPTOR_SET_PER_DRAW,
+        "sDither" |
+        "sRenderTasks" |
+        "sGpuCache" |
+        "sTransformPalette" |
+        "sPrimitiveHeadersF" |
+        "sPrimitiveHeadersI" => return DESCRIPTOR_SET_PER_INSTANCE,
+        _ => unreachable!(),
```

Comment:
Maybe we could add 
```rust
x => unreachable!("Sampler not found: {:?}", x),
```
here, this would gave us a more verbose panic, if a new sampler introduced and we don't handle it.